job_templates/module_action_-_ansible_default.erb

```diff
@@ -0,0 +1,50 @@
+<%#
+kind: job_template
+name: Module Action - Ansible Default
+model: JobTemplate
+job_category: Modules
```

Comment:
Unfortunately the category name has to be "Ansible Modules" to keep conventions with everything else.  Every category only has one job template :-\  There's a BZ open to change that.


![screenshot from 2018-08-10 13-00-24](https://user-images.githubusercontent.com/429763/43971153-0a3d8824-9c9e-11e8-9995-325c458d6fbc.png)